docs/src/getting_started.md

```diff
@@ -0,0 +1,146 @@
+# Getting Started with Chmy.jl
+
+[Chmy.jl](https://github.com/PTsolvers/Chmy.jl) is powered by [KernelAbstractions.jl](https://github.com/JuliaGPU/KernelAbstractions.jl) and it is a backend-agnostic toolkit for finite difference computations on multi-dimensional computational staggered grids. In this introductory tutorial, we will show the essence of Chmy.jl in which we resolved a simple 2D diffusion problem. The full code of the tutorial material is available under [diffusion_2d.jl](https://github.com/PTsolvers/Chmy.jl/blob/main/examples/diffusion_2d.jl).
+
+
+## Basic Diffusion
+
+The diffusion equation is a second order parabolic PDE, here for a multivariable function $C(x,t)$ that represents the temperature field showing derivatives in both temporal $\partial t$ and spatial $\partial x$ dimensions, where $D$ is the diffusion coefficient. In 2D we have the following formulation for the diffusion process:
+
+```math
+\begin{equation}
+\frac{\partial C}{\partial t} = D \left( \frac{\partial^2 C}{\partial x^2} + \frac{\partial^2 C}{\partial y^2} \right).
+\end{equation}
+```
+
+Using the Fourier's law of heat conduction, which relates the heat flux $q$, $(W/m^2)$ to the temperature gradient $\frac{\partial C}{\partial x_i}$ $(K/m)$, we can rewrite equation `(1)` as a system of two PDEs, consisting of equations `(2)` and `(3)`.
+
+```math
+\begin{equation}
+q := -D \nabla C,
+\end{equation}
+```
+```math
+\begin{equation}
+\frac{\partial C}{\partial t} = - \nabla \cdot q.
+\end{equation}
+```
+
+### Boundary Condition
+
+Generally, partial differential equations (PDEs) require initial or [boundary conditions](./concepts/bc.md) to ensure a unique and stable solution. For the temperature field `C`, a Neumann boundary condition is given by:
+
+
+```math
+\begin{equation}
+\frac{\partial C}{\partial n} = g(x, t)
+\end{equation}
+```
+
+where $\frac{\partial C}{\partial n}$ is the derivative of `C` normal to the boundary, and $g(x, t)$ is a given function. In this tutorial example, we consider a homogeneous Neumann boundary condition, $g(x, t) = 0$, which implies that there is no flux across the boundary.
+
+
+## Using Chmy.jl for Backend Portable xPU Implementation
+
+As the first step, we need to load the main module and any necessary submodules of [Chmy.jl](https://github.com/PTsolvers/Chmy.jl). Moreover, we use [KernelAbstractions.jl](https://github.com/JuliaGPU/KernelAbstractions.jl) for writting backend-agnostic kernels that are compatible with Chmy.jl. One may also load additional modules for results analysis or plotting etc..
+
+```julia
+using Chmy, Chmy.Architectures, Chmy.Grids, Chmy.Fields, Chmy.BoundaryConditions, Chmy.GridOperators, Chmy.KernelLaunch
+using KernelAbstractions # for CPU or various GPU backend
+using Printf, CairoMakie # for I/O and plotting
+# using AMDGPU
```

Comment:
I would remove this commented line

Reply:
The `using AMDGPU` line? I actually added the description in the lines below, let me @ you in the relevant code section